metadata-ingestion/src/datahub/ingestion/source/delta_lake/source.py

```diff
@@ -122,6 +141,99 @@ def get_fields(self, delta_table: DeltaTable) -> List[SchemaField]:
 
         return fields
 
+    def _create_operation_aspect_wu(
+        self, delta_table: DeltaTable, dataset_urn: str
+    ) -> Iterable[MetadataWorkUnit]:
+        for hist in delta_table.history(self.source_config.version_history_limit):
+
+            # History schema picked up from https://docs.delta.io/latest/delta-utility.html#retrieve-delta-table-history
+            reported_time: int = int(time.time() * 1000)
+            last_updated_timestamp: int = hist["timestamp"]
+            statement_type = OPERATION_STATEMENT_TYPES.get(
+                hist.get("operation"), OperationTypeClass.CUSTOM
+            )
+            custom_type = hist.get("operation")
+            operation_custom_properties = dict()
+            operation_custom_properties.update(
```

Comment:
Can we rewrite this update section to improve readability. Issue update only if value is not None.

Something like
```
if hist.get(key1) is not None:
    properties[key1] = hist[key1]
```

OR
```
keys_interested_in = [key1, key2, ...]
for key in hist.keys():
    if key in keys_interested_in and hist[key] is not None:
        properties[key] = hist[key]
```